types/node/globals.d.ts

```diff
@@ -234,6 +234,20 @@ declare global {
              * Is this a constructor call?
              */
             isConstructor(): boolean;
+
+            getScriptHash(): string;
+            getScriptNameOrSourceURL(): string;
+
+            getScriptNameOrSourceURL(): string;
+            getEnclosingColumnNumber(): number;
+            getEnclosingLineNumber(): number;
+            getPosition(): number;
+
+            getPromiseIndex(): number;
+            isAsync(): boolean;
+            isPromiseAll(): boolean;
```

Comment:
Please add some docs from https://v8.dev/docs/stack-trace-api#customizing-stack-traces

Reply:
docs added for these 3 methods.